imports/plugins/core/email/server/no-meteor/register.js

```diff
@@ -1,5 +1,13 @@
-import mutations from "./mutations";
-import startup from "./startup";
+import { Meteor } from "meteor/meteor";
+
+// This is temporary. Mutations still import jobs, which don't
+// work outside of a Meteor environment.
+let mutations = {};
+let startup = () => {};
+if (!Meteor.isFakeMeteor) {
+  mutations = require("./mutations").default;
```

Comment:
I think it's weird to mix import/export syntax with CommonJS requires. You can dynamically import modules with the `import` function. See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import#Dynamic_Imports for details.

Reply:
I agree but I don't think import as a function is supported yet in Node 8. It may work due to our Babel plugins, but since this is only temporary until no-Meteor conversion is done and I'd have to `await` it, I think I'll stick with this.